api/scripts/certification/fill-issue-report-category-id.js

```diff
@@ -0,0 +1,62 @@
+const logger = require('../../lib/infrastructure/logger');
+const { knex, disconnect } = require('../../db/knex-database-connection');
+const bluebird = require('bluebird');
+
+const isLaunchedFromCommandLine = require.main === module;
+
+async function getCertificationIssueReportIdCategorySubcategory() {
+  return knex('certification-issue-reports').select('id', 'category', 'subcategory');
+}
+
+async function main() {
+  logger.info(`Script ${__filename} est lancé !`);
+
+  const certificationIssueReports = await getCertificationIssueReportIdCategorySubcategory();
+
+  logger.info(`Nb de certification issue reports à modifier : ${certificationIssueReports.length}`);
+
+  const categories = await knex('issue-report-categories').select('name', 'id');
+
+  await _updateIssueReportsWithCategoryId(certificationIssueReports, categories);
+
+  const { count: reportsNotUpdated } = await knex('certification-issue-reports').count('*').whereNull('categoryId');
+
+  if (reportsNotUpdated > 0) {
+    logger.info(
+      `Nb de certification issue reports non mis à jour : ${reportsNotUpdated.length} / ${certificationIssueReports.length}`
+    );
+  } else {
+    logger.info(`${certificationIssueReports.length} certification issue reports mis à jour`);
+  }
+}
+
+(async () => {
+  if (isLaunchedFromCommandLine) {
+    try {
+      await main();
+    } catch (error) {
+      console.error(error);
+      process.exitCode = 1;
+    } finally {
+      await disconnect();
+    }
+  }
+})();
+
+module.exports = { main };
+
+async function _updateIssueReportsWithCategoryId(certificationIssueReports, categories) {
+  await bluebird.map(certificationIssueReports, async (certificationIssueReport) => {
+    const getCategoryId = (lookupName) => categories.find(({ name }) => lookupName === name).id;
+    if (certificationIssueReport.subcategory) {
+      certificationIssueReport.categoryId = getCategoryId(certificationIssueReport.subcategory);
+    } else if (certificationIssueReport.category) {
+      certificationIssueReport.categoryId = getCategoryId(certificationIssueReport.category);
+    }
```

Comment:
Pas besoin de modifier mais on aurait pu faire un peu plus court:
```suggestion
    const category = certificationIssueReport.subcategory ?? certificationIssueReport.category;
    certificationIssueReport.categoryId = getCategoryId(category);
 
```